doc/classes/Image.xml

```diff
@@ -238,51 +253,56 @@
 			<argument index="0" name="rect" type="Rect2">
 			</argument>
 			<description>
-				Return a new [code]Image[/code] that is a copy of "area" in this [code]Image[/code].
+				Returns a new image that is a copy of the image's area specified with [code]rect[/code].
 			</description>
 		</method>
 		<method name="get_size" qualifiers="const">
 			<return type="Vector2">
 			</return>
 			<description>
+				Returns a [code]Vector2[/code] that is the size of the image.
 			</description>
 		</method>
 		<method name="get_used_rect" qualifiers="const">
 			<return type="Rect2">
 			</return>
 			<description>
-				Return the area of this [code]Image[/code] that is used/visibly colored/opaque.
+				Returns a new [code]Rect2[/code] that is the size of the image that is colored and opaque.
```

Comment:
- Repeating "that is"
- Link to `Rect2` class
- opaque meaning only alpha of 1.0, or any pixel with alpha > 0 ? I would have to check the code, but I'm pretty sure it's the latter.

Returns a [Rect2] enclosing the visible portion of the image.